src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_controlnet.py

```diff
@@ -746,20 +858,15 @@ def __call__(
                 latent_model_input = torch.cat([latents] * 2) if do_classifier_free_guidance else latents
                 latent_model_input = self.scheduler.scale_model_input(latent_model_input, t)
 
+                # controlnet(s) inference
                 down_block_res_samples, mid_block_res_sample = self.controlnet(
                     latent_model_input,
                     t,
                     encoder_hidden_states=prompt_embeds,
-                    controlnet_cond=image,
+                    controlnet_conditions=controlnet_conditions,
                     return_dict=False,
                 )
 
-                down_block_res_samples = [
```

Comment:
I like the clean-up here, but I think since we have to pay attention to this: https://github.com/huggingface/diffusers/pull/2627/files#r1132300008 we should maybe put all this directly in the `ControlNetModel` forward call? Also see:

Reply:
That's a nice idea! I'll write it in that direction.

Reply:
Fixed in https://github.com/huggingface/diffusers/pull/2627/commits/d1acef4b7c372f9a1c77f526b0d67f727b81064d